### PR TITLE
Extend event_basics and remove event_posrec_many

### PR DESCRIPTION
**Pull request by jingqiangye**
**What is the problem / what does the code in this PR do**

- Add more properties at the event level, see https://github.com/XENONnT/straxen/issues/381.
- Merge event_posrec_many into eventbasics. The use case of having this datakind separately is considered marginal. It makes little sense to keep the those 12 parameters out of the event_basics if event_basics already has 66 fields. As such, let's make things a bit more simple and delete event_posrec_many.

**Can you briefly describe how it works?**
The added properties will be loaded in `event_basics`.

**Can you give a minimal working example (or illustrate with a figure)?**
```python
import straxen
straxen.get_df(run_id, 'event_basics')
```

### CODE REVIEW

File: straxen/plugins/event_processing.py [hunk 1]
```diff
@@ -126,7 +126,11 @@ class EventBasics(strax.LoopPlugin):
         ('area',              np.float32, 'area, uncorrected [PE]'),
         ('n_channels',        np.int32,   'count of contributing PMTs'),
         ('n_competing',       np.float32, 'number of competing PMTs'),
+        ('max_pmt',           np.int32,   'PMT number which contributes the most PE'),
```
[the remark this answers is not in the document]
Comment (WenzDaniel) [reply straxen/plugins/event_processing.py:129]:
```suggestion
        ('max_pmt',           np.int16,   'PMT number which contributes the most PE'),
```
I checked with `np.iinfo` int16 -> max integer would be 32767 which should be find even for DARWIN.

**Review by WenzDaniel — COMMENTED** [review]
Thank you @jingqiangye for this updated. I quickly processed run `015258` and loaded the new parameters I did not spot any errors. 
I just pushed the version bump. This PR will trigger some reprocessing since we wont be able to load older data due to the dtype changes. @jorana and @ershockley can maybe comment when it would be best to merge this PR.

**Comment by jorana** [comment]
@WenzDaniel, reprocessing is no deal at all since we are talking about Events

**Comment by jingqiangye** [comment]
@WenzDaniel Thanks for the version bump and the PMT number type change! One thing to note is that I didn't add `s2_over_tdiff` since Amanda and Jacques have some idea to replace it with a better parameter. @AmandaDepoian @jpienaar13 are you going to add the new parameter here?

**Comment by WenzDaniel** [comment]
@jingqiangye @AmandaDepoian @jpienaar13 , we would like to make a new release today and I would like to merge this PR too, if you do not disagree. I will wait with the merge until 15:00 CET today. So if you can add `s2_over_tdiff ` by then it would be nice, if not also no problem then we can make a new PR next week.

File: straxen/plugins/event_processing.py [hunk 2]
```diff
@@ -260,7 +267,7 @@ def compute_loop(self, event, peaks):
                     result[f'alt_s{s_i}_{name}'] = secondary_s[s_i][name]
                 if s_i == 2:
                     for name in posrec_save:
-                        posrec_result[f'alt_s{s_i}_{name}'] = secondary_s[s_i][name]
+                        result[f'alt_s{s_i}_{name}'] = secondary_s[s_i][name]
```
[the remark this answers is not in the document]
Comment (jorana) [reply straxen/plugins/event_processing.py:270]: agree but not going to change this now

**Review by jorana — APPROVED** [review]
Thanks Jingqiang. I also removed event_posrec many, having it separately is not really worth it if event_basics is 66 fields and we are just keeping this 12 fields out of it seems to be a little bit overly complex.

**Pull request merged**